resources/virt.go

```diff
@@ -49,18 +62,21 @@ type VirtRes struct {
 	Transient  bool               `yaml:"transient"` // defined (false) or undefined (true)
 	CPUs       uint16             `yaml:"cpus"`
 	Memory     uint64             `yaml:"memory"` // in KBytes
+	OSInit     string             `yaml:"osinit"` // init used by lxc
```

Comment:
Well OSInit needs to be replaced anyway, since the parameters need to be passed separately (or we would have to parse them)

I think this is too extensive for this first attempt, I wouldn't call the virt resource finished though